src/commonTest/kotlin/com/charleskorn/kaml/YamlWritingTest.kt

```diff
@@ -885,6 +886,23 @@ class YamlWritingTest : DescribeSpec({
                 }
             }
         }
+
+        describe("handling comments") {
+            context("comments in Java object") {
+                val input = SimpleStructureWithComments("objName", 73, "justTest")
+
+                it("is written") {
+                    Yaml.default.encodeToString(SimpleStructureWithComments.serializer(), input) shouldBe """
+                        name: "objName"
+                        #Cool int
```

Comment:
Should there be a space between the `#` and the comment content perhaps? Or should this be configurable? What do you think?

Reply:
Not sure. Like an additional `@YamlComment` annotation parameter `prefix`? I don't think it's needed but I can add it if you want

Reply:
I think it's fairly safe to default to adding a space - if anyone asks for something different in the future, we can make it configurable.